runtime/coverage.go

```diff
@@ -120,6 +120,9 @@ type CoverageReport struct {
 	// This filter can be used to inject custom logic on
 	// each location/program inspection.
 	LocationFilter LocationFilter `json:"-"`
+	// Contains a mapping with human-friendly names for
+	// locations. This could be the filepath of a location.
+	LocationMappings map[string]string `json:"-"`
```

Comment:
Just some general Qs on the `CoverageReport` struct:
- Do we need to export these fields?
- I think this coverage report struct is not marshaled as JSON as-is, correct? If so, we could remove these JSON marshaling tags. i.e: `json:"-"`

Reply:
Good catch :+1: 
Updated in https://github.com/onflow/cadence/pull/2859/commits/4871d50d108d715a582746143ff09d72e78c525b

Reply:
I have left the necessary fields because the coverage report struct is indeed marshaled as JSON in https://github.com/onflow/cadence/blob/master/runtime/coverage.go#L433-L441

Reply:
> I have not touched the necessary fields because the coverage report struct is marshaled as JSON in

But aren't all the fields of `CoverageReport` have the "ignore-tag" (i.e: `json:"-"`), and thus all the fields are skipped by the marshaller?

Reply:
Ah, you are right :clap: 
For some reason I was using a redundant type alias for `CoverageReport`, which was also marshaled.
Fixed in https://github.com/onflow/cadence/pull/2859/commits/6b9d9f2fcd558ec316794b345bb8d9a9aadd0525

Reply:
nice! 👍